spec/models/user_spec.rb

```diff
@@ -13,4 +13,24 @@
     end
   end
 
+ describe "#add_curators_as_proxies" do
+
+    let(:curatorA) { FactoryBot.create(:user) }
+    let(:curatorB) { FactoryBot.create(:user) }
+
+    before do
+      # stub group service implementation to return curator group members
+      allow(RoleMapper).to receive(:map).and_return({ "curator" => [ curatorA.uid, curatorB.uid ]})
```

Comment:
Minor, picky point.  Since User.curators now calls RoleMapper.whois rather than RoleMapper.map, it might be best to change the line above to reflect that ... i.e.,
```ruby
allow(RoleMapper).to receive(:whois).with(User::CURATOR_GROUP).and_return([ curatorA.uid, curatorB.uid ])
```
I wouldn't hold up the merge if you'd prefer to leave it the way it is, though.